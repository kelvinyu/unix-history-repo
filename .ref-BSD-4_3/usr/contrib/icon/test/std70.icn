record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("1 >= 1 ----> ",image(1 >= 1) | "none")
   write("1 >= 2 ----> ",image(1 >= 2) | "none")
   write("1 >= 0 ----> ",image(1 >= 0) | "none")
   write("-1 >= 0 ----> ",image(-1 >= 0) | "none")
   write("1 >= -2 ----> ",image(1 >= -2) | "none")
   write("-1 >= -0 ----> ",image(-1 >= -0) | "none")
   write("1. >= 1 ----> ",image(1. >= 1) | "none")
   write("1 >= 2. ----> ",image(1 >= 2.) | "none")
   write("1. >= 0. ----> ",image(1. >= 0.) | "none")
   write("-1 >= 0. ----> ",image(-1 >= 0.) | "none")
   write("1. >= -2 ----> ",image(1. >= -2) | "none")
   write("-1 >= -0. ----> ",image(-1 >= -0.) | "none")
   write("2 > \"a\" ----> ",image(2 > "a") | "none")
end
