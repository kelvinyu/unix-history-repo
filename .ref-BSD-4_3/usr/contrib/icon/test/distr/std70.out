1 >= 1 ----> 1
1 >= 2 ----> none
1 >= 0 ----> 0
-1 >= 0 ----> none
1 >= -2 ----> -2
-1 >= -0 ----> none
1. >= 1 ----> 1.0
1 >= 2. ----> none
1. >= 0. ----> 0.0
-1 >= 0. ----> none
1. >= -2 ----> -2.0
-1 >= -0. ----> none
Run-time error 102 at line 21 in std70.icn
numeric expected
offending value: "a"
